augur/curate/titlecase.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""
+Standardizes string fields of the NDJSON record from stdin and outputs the
+modified record to stdout.
+"""
+import argparse
+
+import re
+import unicodedata
+from typing import Optional, Set, Union
+
+from augur.errors import AugurError
+from augur.io.print import print_err
+from augur.types import DataErrorMethod
+
+def register_parser(parent_subparsers):
+    parser = parent_subparsers.add_parser("titlecase",
+    parents = [parent_subparsers.shared_parser],
+    help = __doc__)
+    
+    required = parser.add_argument_group(title="REQUIRED")
+    required.add_argument("--titlecase-fields", nargs="*",
+        help="List of fields to convert to titlecase.", required=True)
+
+    optional = parser.add_argument_group(title="OPTIONAL")
+    optional.add_argument("--articles", nargs="*",
+        help="List of articles that should not be cast to titlecase.")
+    optional.add_argument("--abbreviations", nargs="*",
+        help="List of abbreviations that should not be cast to titlecase, keeps uppercase.")
```

Comment:
s/cast/converted/ to be consistent with the other descriptions.

Reply:
Thanks, fixup in https://github.com/nextstrain/augur/pull/1197/commits/c00156ba7d31026aa89fc5ed68ae5215c09b99bd